policyengine_us/variables/gov/states/ca/tax/income/credits/earned_income/ca_eitc.py

```diff
@@ -89,6 +89,17 @@ def modify_parameters(parameters: ParameterNode) -> ParameterNode:
         return simulation.branches.get("ca_eitc")
 
 
+class ca_eitc_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for CalEITC"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        branch = get_ca_eitc_branch(tax_unit, period, parameters)
+        return branch.calculate("eitc_eligible", period)
+
+
 class ca_eitc(Variable):
```

Comment:
I think we could now add `defined_for = "ca_eitc_eligible"`. @nikhilwoodruff any issues with `defined_for` + branching?